test/test_sox_compatibility.py

```diff
@@ -251,29 +253,26 @@ def test_bass(self):
         q = 0.707
         gain = 40
 
-        noise_filepath = common_utils.get_asset_path('whitenoise.wav')
         E = torchaudio.sox_effects.SoxEffectsChain()
-        E.set_input_file(noise_filepath)
+        E.set_input_file(self.noise_filepath)
         E.append_effect_to_chain("bass", [gain, central_freq, str(q) + 'q'])
         sox_output_waveform, sr = E.sox_build_flow_effects()
 
-        waveform, sample_rate = torchaudio.load(noise_filepath, normalization=True)
+        waveform, sample_rate = torchaudio.load(self.noise_filepath, normalization=True)
         output_waveform = F.bass_biquad(waveform, sample_rate, gain, central_freq, q)
 
-        self.assertEqual(output_waveform, sox_output_waveform, atol=1.5e-4, rtol=1e-5)
+        self.assertEqual(output_waveform, sox_output_waveform, atol=1e-3, rtol=1e-4)
```

Comment:
This is dues to edge effect, and the trick to avoid reducing tolerance here is to scale the intensity of the generated whitenoise (say `x0.9`).

Reply:
Tried this was `bass` and still got a few errors exceeding 1e-3.
```
E   AssertionError: False is not true : Tensors failed to compare as equal! With rtol=0.0001 and atol=0.001, found 136 element(s) (out of 220500) whose difference(s) exceeded the margin of error (including 0 nan comparisons). The greatest difference was 0.0013254880905151367 (0.8861521482467651 vs. 0.88482666015625), which occurred at index (0, 47224).
```